foundation/foundation-mda/src/main/resources/templates/deployment/spark-infrastructure/v2/spark.infrastructure.values.yaml.vm

```diff
@@ -42,6 +42,16 @@ aissemble-thrift-server-chart:
      </configuration>
 
 aissemble-hive-metastore-service-chart:
+  mysql:
+    auth:
+      # Note: Changing these values requires removal of the `hive-metastore-db-0` PVC, or manual modification of the
+      # persisted database.
+      username: hive
+      # NB: UCS
```

Comment:
I: Flagging for future improvement as we roll out the UCS.  This chart will not be active by default until improved via UCS.